src/utils/RTEvaluation.cpp

```diff
@@ -86,6 +86,8 @@ class TOPPIdXMLEvaluation :
   {
     registerInputFile_("in", "<file>", "", "Input file");
     setValidFormats_("in", ListUtils::create<String>("idXML"));
+    registerInputFile_("sequences_file", "<file>", "", "Fasta File", false);
+    setValidFormats_("in", ListUtils::create<String>("fasta"));
```

Comment:
I think the valid formats should be set on sequence_file parameter and not on “in”

Reply:
sure